reference/api/keys.md

```diff
@@ -182,80 +237,84 @@ Get information on the specified key. Attempting to use this endpoint with a non
 }
 ```
 
-For an explanation of these fields, see the [get all keys endpoint](#returned-fields).
+For an explanation of these fields, see the [key object](#key-object).
 
 ## Create a key
 
 <RouteHighlighter method="POST" route="/keys"/>
 
 Create an API key with the provided description, permissions, and expiration date.
 
-Only the `indexes`, `actions`, and `expiresAt` fields are mandatory.
-
 ### Body
 
 #### `name`
 
-**Type:** string
+**Type:** String
+
 **Default value:** `null`
 
 A human-readable name for the key.
 
 #### `uid`
 
-**Type:** string
-**Default value:** none
+**Type:** String
+
+**Default value:** N/A
 
 A [uuid v4](https://www.sohamkamani.com/uuid-versions-explained) to identify the API key. If not specified, it is generated by Meilisearch.
 
 #### `description`
 
-**Type:** string
+**Type:** String
+
 **Default value:** `null`
 
 An optional description for the key.
 
-#### `actions`
+#### `actions` *
```

Comment:
>Do we need to specify that the * means required?

I think most users would know as many API references use it, but it might be better to specify it. We could mention it in the overview, so we don't have to mention it on every page

Reply:
Let's assume that users can understand it for now, and if we get a complaint about it we can clarify at that time.